scripts/ida_toggles.py

```diff
@@ -27,7 +27,21 @@ def __init__(self, name, configuration=None):
         self.annotation_report_path = None
         self.configuration = configuration if configuration else {}
 
-    def add_toggle_data(self, state_data_path):
+    def get_toggles_data_summary(self):
+        data = []
+        for toggle_type, toggles in self.toggles.items():
+            for toggle_name, toggle in toggles.items():
+                data.append(toggle.get_summary_report())
+        return data
+
+    def get_full_report(self):
+        data = []
+        for toggle_type, toggles in self.toggles.items():
+            for toggle_name, toggle in toggles.items():
+                data.extend(toggle.get_full_reports())
+        return data
```

Comment:
I thought Régis had provided feedback about switching to values when you aren't using the keys.  Maybe that was elsewhere?

Reply:
He did, but it was elsewhere